### PR TITLE
Fix #1798

### LINKED ISSUE #1798: Leaderboard Table is not horizontally scrollable in Mobile View
The leaderboard table on the path on challenge page is not horizontally scrollable in mobile view.
(see example on "https://evalai.cloudcv.org/web/challenges/challenge-page/103/leaderboard/298")

Screenshot:

<div>

</div>
<br><br>
Can I work on this ?

### PR DESCRIPTION
**Pull request by TejasBhitle**
Fix #1798 : Leaderboard table is now horizontally scrollable in mobile view".

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@TejasBhitle Can you please check why the travis build is failing?

File: frontend/src/views/web/challenge/leaderboard.html [hunk 1]
```diff
@@ -35,7 +35,8 @@
                         <div class="horizontal-scroll">
                             <div class="col s12">
                                 <div ng-if="!challenge.isResult" class="result-wrn">No phase selected.</div>
-                                <table ng-if="challenge.isResult && challenge.leaderboard.length" class="highlight">
+                                <table ng-if="challenge.isResult && challenge.leaderboard.length" class="highlight"
+                                       style="display: block; overflow-x: auto; white-space: nowrap;">
```
Comment (RishabhJain2018) [inline frontend/src/views/web/challenge/leaderboard.html]: Please don't use inline CSS.

**Comment by TejasBhitle** [comment]
@RishabhJain2018 I guess the travis build was failing because to initialise the sqs queue, AWS_ACCESS_KEY_ID & AWS_SECRET_ACCESS_KEY were need to be set as an environment variable which are set from the repository settings in travis only.

Can you please review the PR ?

File: frontend/src/views/web/challenge/leaderboard.html (hunk 1 above)
Comment (TejasBhitle) [reply frontend/src/views/web/challenge/leaderboard.html]: Ok let me fix that

**Comment by TejasBhitle** [comment]
@RishabhJain2018 I have removed inline css as requested. Please review. Thanks

**Comment by gautamjajoo** [comment]
Closing this since it is already solved in https://github.com/Cloud-CV/EvalAI/pull/1845.

**Pull request closed without merging**